File: C++/Conditional Statements /Code3.cpp
//Print if a number is Even or Odd
#include<iostream>
using namespace std;

int main(){
    int n;
    cout << "Enter your number = ";
    cin >> n;

    if(n % 2 == 0){
        cout << "The number is Even" << endl;
    }
    else{
        cout << "The number is Odd" << endl;
    }
    return 0;
}

File: Java/Patterns (Part - 1)/Code3.java
import java.util.Scanner;

public class Code3 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.print("Enter your choice of number of lines = ");
        int a = sc.nextInt();

        for(int line = 1; line <= a; line++){
            for(int i = 1; i <= line; i++){
                System.out.print(i);
            }
            System.out.println();
        }
        sc.close();
    }
}
